docs/Troubleshooting.md

```diff
@@ -93,3 +93,12 @@ If you run into issues where running `react-native init` hangs in your system, t
 ```
 react-native init --verbose
 ```
+
+## Unable to start react-native package manager
```

Comment:
Done but Circle CI doesn't pass, don't know why... Can you take a look? (I just added "on Linux"...)